### PR TITLE
Added release notes and reviewed content

### PR DESCRIPTION
**Pull request by Vic-Cooper**
**Select the type of change:**

- [ ] Small Changes - Typos, formatting, slight revisions
- [x ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site and Docusaurus

**Purpose of the Pull Request:**
Added missing release notes for versions 1.3.0-pre.3 to 1.3.2.

**Issue Number:**
[](https://jira.unity3d.com/browse/DOCF-6441)

### CODE REVIEW

File: mppm/release-notes/release-notes.md [hunk 1]
```diff
@@ -12,14 +12,14 @@ description: Release notes for Multiplayer Play Mode including new features, upd
 - Fixed the compilation error "'UnityPlayer' does not contain a definition for 'Role'" when the Dedicated Server package version doesn't match.
 
 ### Added
-- Avoid reconstructing virtual player windows through Play modes if their layouts in PlayMode and EditMode are the same. This helps retain anchoring in Windows.
+- The Virtual Player window now keeps the same layout when you switch between Play mode and Edit mode.
```
Comment (gregk-unity) [inline mppm/release-notes/release-notes.md:15]: It's specific to windows so I think it's worth mentioning it.. so more like "In Windows, the Virtual Player window now retains the anchoring when the same layouts are used when you switch between Play mode and Edit mode."

in other words the original description made a little more sense. it's the anchoring or window position that persists and not the layouts.

Comment (Vic-Cooper) [reply mppm/release-notes/release-notes.md:15]: Thanks! :)

File: mppm/release-notes/release-notes.md [hunk 2]
```diff
@@ -4,6 +4,98 @@ title: Release notes
...
 
+## [1.3.2] - 2024-11-14
+
+### Fixed
+- Fixed clones so that they correctly deactivate when you stop a play mode scenario during the clone activation.
+- Fixed the clone editor's layout configuration so that it displays correctly when you enable Playmode Tools.
+- Fixed the compilation error "'UnityPlayer' does not contain a definition for 'Role'" when the Dedicated Server package version doesn't match.
+
+### Added
+- On Windows devices, the Virtual Player window now reamins anchored when you use the same layout in Play mode and Edit mode.
```
Comment (gregk-unity) [inline mppm/release-notes/release-notes.md:15]: typo: reamins

Comment (Vic-Cooper) [reply mppm/release-notes/release-notes.md:15]: And this is why we proofread :D

**Review by gregk-unity — APPROVED** [review]

**Pull request merged**